# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/ModuleLoadRequest.cpp
# Commit: b8ecf23519c9
# Full Hash: b8ecf23519c9ab5180d08f00c712674bfe968dac
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-21 21:39:46
# Regressor Bug: 1742437
# File Overlap Count: 5
# Description:
#   Bug 1742437 - re-scope all generic module/script loader files under js/loader; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138133
# ==============================================================================

diff -r 85fd9fc9cffa -r b8ecf23519c9 js/loader/ModuleLoadRequest.cpp
--- a/js/loader/ModuleLoadRequest.cpp	Mon Feb 21 15:39:08 2022 +0000
+++ b/js/loader/ModuleLoadRequest.cpp	Mon Feb 21 15:39:09 2022 +0000
@@ -11,8 +11,7 @@
 #include "LoadedScript.h"
 #include "ModuleLoaderBase.h"
 
-namespace mozilla {
-namespace dom {
+namespace JS::loader {
 
 #undef LOG
 #define LOG(args)                                                           \
@@ -55,10 +54,10 @@
 
 ModuleLoadRequest::ModuleLoadRequest(
     nsIURI* aURI, ScriptFetchOptions* aFetchOptions,
-    const SRIMetadata& aIntegrity, nsIURI* aReferrer,
-    ScriptLoadContext* aContext, bool aIsTopLevel, bool aIsDynamicImport,
-    ModuleLoaderBase* aLoader, VisitedURLSet* aVisitedSet,
-    ModuleLoadRequest* aRootModule)
+    const mozilla::dom::SRIMetadata& aIntegrity, nsIURI* aReferrer,
+    mozilla::dom::ScriptLoadContext* aContext, bool aIsTopLevel,
+    bool aIsDynamicImport, ModuleLoaderBase* aLoader,
+    VisitedURLSet* aVisitedSet, ModuleLoadRequest* aRootModule)
     : ScriptLoadRequest(ScriptKind::eModule, aURI, aFetchOptions, aIntegrity,
                         aReferrer, aContext),
       mIsTopLevel(aIsTopLevel),
@@ -192,5 +191,4 @@
   mDynamicPromise = nullptr;
 }
 
-}  // namespace dom
-}  // namespace mozilla
+}  // namespace JS::loader